tests/TransactionTest.php

```diff
@@ -332,7 +332,7 @@ public function testTransaction(): void
         $count = User::count();
         $this->assertEquals(2, $count);
 
-        $this->assertTrue(User::where('alcaeus')->exists());
+        $this->assertTrue(User::where('name', 'alcaeus')->exists());
```

Comment:
Being too permissive didn't allow us to detect this incorrect query.

```diff
- { "alcaeus": null }
+ { "name": "alcaeus" }
```

The new exception made it surface.